specification/graphrbac/data-plane/Microsoft.GraphRbac/stable/1.6/graphrbac.json

```diff
@@ -387,7 +387,7 @@
             "in": "body",
             "required": true,
             "schema": {
-              "$ref": "#/definitions/ApplicationUpdateParameters"
+              "$ref": "#/definitions/Application"
```

Comment:
That's not a good idea, I designed this on purpose to separate stuff which are "patchable" from the GET model "Application"

Reply:
Done, but one thing though, the [api spec](https://docs.microsoft.com/en-us/previous-versions/azure/ad/graph/api/entity-and-complex-type-reference#application-entity) you mentioned is a bit old, e.g. `publicClient` is actually patch'able which CLI has leveraged to create a native client. Otherwise, I have made the adjustment you have asked. I keep the `ApplicationUpdateParameters` for backward compatible.